ossdbtoolsservice/language/operations_queue.py

```diff
@@ -8,11 +8,11 @@
 import threading
 from logging import Logger  
 from queue import Queue
+from prompt_toolkit.completion import Completer
```

Comment:
Sort them alphabetically

Reply:
Should I sort by prompt_toolkit or Completer?